### DISCUSSION TITLE
I would like to ask if milvus provides an LRU（Least Recently Used） method? How to use it?

### DISCUSSION THREAD

**Discussion by LMERose** [milvus-io/milvus#33088] [Q&A and General discussion]
Hello everyone, I would like to ask if milvus provides an LRU（Least Recently Used） method? How to use it?

**Comment by yhmo** [comment]
I suppose LRU for loading data is a new ability of v2.5.
Typically, this ability is an internal machinery of the loading process, users no need to care about it.

**Reply by LMERose** [reply]
Thank you for your reply. Because the collection I built is large, it takes a lot of time to load the previous data every time I use it, so I was looking for a way to improve the loading time. Or solve the situation where there is a limit on memory resources, but the amount of data is large.

**Reply by xiaofan-luan** [reply]
if you have local nvme ssd, we would recommend to enable mmap feature or use diskann.
this might be much slower than purely memory index but could load data significantly larger
